app/budget/directives/courseCosts/courseCosts.html

```diff
@@ -34,23 +34,87 @@
 					</td>
 					<!-- Enrollment -->
 					<td class="course-costs__table-cell">
-						{{ sectionGroup.totalSeats }}
+						<div class="course-costs__edit-container">
+							<div class="course-costs__number-input">
+								<ipa-input
+									mode="'number'"
+									value="sectionGroup.overrideTotalSeats"
+									on-update="overrideSectionGroup(sectionGroup, 'seats')">
+								</ipa-input>
+							</div>
+							<div class="glyphicon glyphicon-repeat course-costs__revert-ui"
+								uib-tooltip="Revert to schedule total seats of {{ !!(sectionGroup.totalSeats) ? sectionGroup.totalSeats : 'none' }}"
```

Comment:
These should line up with 'class' but you are using indents, so they won't depending on the text editor.